libafl/src/monitors/mod.rs

```diff
@@ -643,7 +68,13 @@ impl Monitor for NopMonitor {
     }
 
     #[inline]
-    fn display(&mut self, _event_msg: &str, _sender_id: ClientId) {}
+    fn display(
```

Comment:
maybe
```
    fn display(
        &mut self,
        _client_stats_manager: &mut ClientStatsManager,
        _event_msg: &str,
        _sender_id: ClientId,
    );
```
i think default implementation that does nothing is not good

Reply:
This is the implementation for NopMonitor, not trait method definition.

Reply:
ah ok sorry.